docs/src/pages/index.tsx

```diff
@@ -1,37 +1,259 @@
 import clsx from 'clsx';
+import {
+  FaChrome,
+  FaCode,
+  FaFile,
+  FaFont,
+  FaGithub,
+  FaImage,
+  FaLink,
+  FaMicrochip,
+} from 'react-icons/fa6';
+import { MdLiveTv } from 'react-icons/md';
 import Link from '@docusaurus/Link';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import Layout from '@theme/Layout';
 import Heading from '@theme/Heading';
 
 import styles from './index.module.css';
+import { PropsWithChildren } from 'react';
+import { IconType } from 'react-icons';
+import MembraneLogo from '@site/static/img/membrane-logo.svg';
+import WebGpuLogoDark from '@site/static/img/webgpu-dark.svg';
+import WebGpuLogoLight from '@site/static/img/webgpu-light.svg';
+import { useColorMode } from '@docusaurus/theme-common';
 
 function HomepageHeader() {
-  const { siteConfig } = useDocusaurusContext();
   return (
     <header className={clsx('hero hero--primary', styles.heroBanner)}>
       <div className="container">
-        <Heading as="h1" className="hero__title">
-          {siteConfig.title}
-        </Heading>
-        <p className="hero__subtitle">{siteConfig.tagline}</p>
-        <div className={styles.buttons}>
-          <Link className="button button--secondary button--lg" to="/docs/intro">
-            Docs
-          </Link>
+        <div className="row">
+          <div className="col col--6">
+            <div className="container">
+              <Heading as="h2" className="hero__title">
+                The best solution for programmatic live video compositing.
+              </Heading>
+              <p>
+                Process live video streams in real-time. Apply effects, combine videos, or add
+                transitions to produce any output you want.
+              </p>
+              <div className="row">
+                <Link
+                  className="button button--secondary button--lg col col-6 margin--sm"
+                  to="/docs/intro">
+                  Docs
+                </Link>
+                <Link
+                  className="button button--secondary button--outline button--lg col col-6 margin--sm"
+                  to="https://github.com/membraneframework/video_compositor">
+                  View on GitHub
+                </Link>
+              </div>
+            </div>
+          </div>
+          <div className="col col--6">Image</div>
```

Comment:
Is it just a temporary placeholder?

Reply:
![87wh9n](https://github.com/membraneframework/video_compositor/assets/9753141/4da7f21d-4559-4dbd-89a0-88dcc03ea09b)